src/apis/api.ts

```diff
@@ -14,3 +15,13 @@ export const userInfo = async (id: number) => {
   const { data } = await instance.get(`/user/${id}`);
   return data;
 };
+
+export const createBoard = async ({ title, content, imgUrl }: ICreateBoard) => {
+  const { data } = await instance.post('/post', { title, content, imgUrl });
+  return data;
+};
+
+export const readBoard = async () => {
```

Comment:
```suggestion
export const getBoardList = async () => {
```

readBoard라고 두루뭉실하게 네이밍을 지으면 개발자는 함수명만 보고 판단이 어려워져 개발자 경험에 좋지 않을 것 같아요 함수명은 명시적일수록 좋다고 생각합니다 다음과 같이 명시적인 네이밍을 고려해 보면 좋을 것 같아유

Reply:
앞으로는 모든 사람들이 쉽게 읽을 수 있도록 네이밍 하겠습니다.